app/status/views.py

```diff
@@ -11,14 +12,16 @@ def status_no_db():
     try:
         return jsonify(
             status="ok",
-            version=utils.get_version_label(),
-            db_version=utils.get_db_version()
+            version=get_version_label(),
+            db_version=utils.get_db_version(),
```

Comment:
No that's fine.  I was confused between utils and dmutils, thought they were the same thing... not having a good day for following what's going on, sorry!